file-upload-tutorial.md

```diff
@@ -0,0 +1,25 @@
+# A tutorial on doing file uploads in Hapi
+
+### Why file uploads
+
+Every website needs content, but not every website needs to create its own content. If you want your users to be able to share images, music or video with their friends and with the world, you have to crack file uploads.
+
+### What we made
+
+We have built a simple website using the Hapi framework for Node.js. It provides a form for the users that allows them to submit files to the server. The server then saves that in a 'pix' directory (though the files do not have to be pictures!).
+
+To aid the rest of our FAC cohort, we have developed a repo that provides a tutorial on building the site.
```

Comment:
'Unexplained' FAC abbreviation for external audience. Also, I would add the word 'github' infront of repo